packages/actor-context-preprocess-query-source-identify/lib/ActorContextPreprocessQuerySourceIdentify.ts

```diff
@@ -125,13 +125,10 @@ export interface IActorContextPreprocessQuerySourceIdentifyArgs extends IActorCo
    * @default {100}
    */
   cacheSize: number;
-  /* eslint-disable max-len */
   /**
    * An actor that listens to HTTP invalidation events
-   * @default {<default_invalidator> a <npmd:@comunica/bus-http-invalidate/^4.0.0/components/ActorHttpInvalidateListenable.jsonld#ActorHttpInvalidateListenable>}
```

Comment:
I don't see the need to modify this to solve #1526.

If you want to change this, let's discuss this first separately.
But it seems unrelated to this PR.